ui/tests/e2e/test.js

```diff
@@ -39,6 +39,36 @@ describe("End-to-end", () => {
     expect(grayDiv).toBeTruthy();
   });
 
+  test("Export to Saturn", async () => {
```

Comment:
opt: Change [beforeAll](https://jestjs.io/docs/en/api#beforeallfn-timeout) to [beforeEach](https://jestjs.io/docs/en/api#beforeeachfn-timeout), and remove the first two lines from your tests.

Reply:
Oops, I did that in the VM and didnt do it locally.